api/applications/serializers.py

```diff
@@ -8,31 +8,34 @@
 
 class ApiOAuth2ApplicationSerializer(JSONAPISerializer):
     """Serialize data about a registered OAuth2 application"""
-    client_id = ser.CharField(help_text="The client ID for this application (automatically generated)",
+    id = ser.CharField(help_text='The client ID for this application (automatically generated)',
+                       read_only=True,
```

Comment:
I'm not sure what you mean about OPTIONS requests. Just to clarify- the field in the database model is actually called `client_id` (the database `_id` does not get exposed to the world). 

Does that affect the name I use for the label?